src/lib/auth/permissions.js

```diff
@@ -0,0 +1,3 @@
+export const viewDropBox = "view:drop_box";
+export const ingestDropBox = "ingest:drop_box";
```

Comment:
Looking at how it is being used, I would rename this permission to use the "upload" verb instead of "ingest", given that we already use the term "ingestion" for dataset tables